projects/valtimo/dossier/src/lib/dossier-list/dossier-list.component.ts

```diff
@@ -64,14 +68,18 @@ moment.locale(localStorage.getItem('langKey') || '');
 export class DossierListComponent implements OnInit {
   @ViewChild('processStartModal') processStart: DossierProcessStartModalComponent;
 
+  public dossierVisibleTabs: Array<DossierListTab> | null = null;
+  public currentCaseType = 'MINE';
```

Comment:
Used?

Reply:
`dossierVisibleTabs` is used to get the tabs from de configService and display it
```
(*ngFor="let tab of dossierVisibleTabs")
```